### PR TITLE
Create if-none-match tag

### PR DESCRIPTION
**Pull request by vilit1**
Replace logic for replace tag with if-none-match

---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to the IoT extension!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] If introducing new functionality or modified behavior, are they backed by unit and integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** unit **and** integration tests passed locally? i.e. `pytest <project root> -vv`
- [ ] Have static checks passed using the .pylintrc and .flake8 rules? Look at the CI scripts for example usage.
- [ ] Have you made an entry in HISTORY.rst which concisely explains your feature or change?

### CODE REVIEW

File: azext_iot/digitaltwins/params.py [hunk 1]
```diff
@@ -287,9 +287,9 @@ def load_digitaltwins_arguments(self, _):
             help="The path to the DTDL component.",
         )
         context.argument(
-            "replace",
-            options_list=["--replace"],
-            help="Indicates the operation should replace an existing twin if it exists."
+            "if_none_match",
+            options_list=["--if-none-match"],
+            help="Indicates the create operation should fail an existing twin if it exists."
```
Comment (digimaun) [inline azext_iot/digitaltwins/params.py:292]: We can make this a bit more clear: "Indicates the create operation should fail if an existing twin with the same id exists."

**Comment by digimaun** [comment]
Let's add at least one command example & description for each command that can make use of `--if-none-match` or `--etag`

https://github.com/vilit1/azure-iot-cli-extension/blob/if_none_match/azext_iot/digitaltwins/_help.py

**Comment by digimaun** [comment]
Let's add an integration test to exercise --if-none-match for `az dt twin relationship create`

**Comment by vilit1** [comment]
image

**Review by digimaun — APPROVED** [review]
